### PR TITLE
Task/numbers

### PR DESCRIPTION
**Pull request by aharin90**

### CODE REVIEW

File: site/specs/numbers.json [hunk 1]
```diff
@@ -4016,39 +4016,38 @@
...
             },
-            "example": null
+            "example": 1234567
           }
         ],
         "requestBody": {},
         "responses": {
           "200": {
-            "description": "<p>Successful response <br> The \"Links\" element includes two links.  The link called \"first\" is the link to the first set of results for the query that was submitted.  If there are more than \"Size\" results, a link called \"next\" is included to help you retrieve the next set of results up to \"Size\" elements. If the results of the query, filtered by the query parameters, do not return any results, the payload will contain empty tag \"EmergencyNotificationRecipients\"</p>\n",
+            "description": "Successful response <br> The \"Links\" element includes two links.  The link called \"first\" is the link to the first set of results for the query that was submitted.  If there are more than \"Size\" results, a link called \"next\" is included to help you retrieve the next set of results up to \"Size\" elements. If the results of the query, filtered by the query parameters, do not return any results, the payload will contain empty tag \"EmergencyNotificationRecipients\"\n",
```
[the remark this answers is not in the document]
Comment (mrmeyers99) [reply site/specs/numbers.json:4031]: You could also you `\`first\`` instead of `\"first\"` which would format it nicely

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: site/specs/numbers.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (aharin90) [reply site/specs/numbers.json:4031]: tried to change " to ` - but it looks bad, so I changed it back

**Review by ajrice6713 — APPROVED** [review]

**Comment by mrmeyers99** [comment]
In general I would like to see `\`` instead of `"` for referencing field names, but this is something we could change later I guess.

File: site/specs/numbers.json [hunk 2]
```diff
@@ -4000,45 +4000,45 @@
...
                   }
                 }
               }
             }
           },
           "400": {
-            "description": "<p>Bad request <br> <p><b>Condition:</b> Unrecognized query parameter <br> <b>Error Description:</b>Query parameter xxx is not recognized. </p> <br> <p><b>Condition:</b>Unsupported value for query parameter <br> <b>Error Description:</b>Invalid value xxx for query parameter yyy. Supported values are aaa, bbb, etc. </p></p>\n",
+            "description": "<p>Bad request </p><br> <p><b>Condition:</b> Unrecognized query parameter <br> <b>Error Description:</b>Query parameter xxx is not recognized. </p> <br> <p><b>Condition:</b>Unsupported value for query parameter <br> <b>Error Description:</b>Invalid value xxx for query parameter yyy. Supported values are aaa, bbb, etc. </p>",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:4022]: does this display properly with the nested `<p>` tags?

**Comment by mrmeyers99** [comment]
@aharin90 can you respond to comments and resolve the conflict with the target branch?

**Review by mrmeyers99 — APPROVED** [review]

**Pull request merged**